data/Store.js

```diff
@@ -74,25 +83,62 @@ export class Store {
      * This is to maximize the ability of downstream consumers (e.g. ag-Grid) to recognize Records
      * that have not changed and do not need to be re-evaluated / re-rendered.
      *
+     * Summary data can be provided via `rawSummaryData` or as the root data if the Store was
+     * created with the treatRootAsSummary flag set to true.
+     *
      * @param {Object[]} rawData
+     * @param {Object} rawSummaryData
      */
     @action
-    loadData(rawData) {
+    loadData(rawData, rawSummaryData) {
+        const {_treatRootAsSummary} = this;
+        throwIf(_treatRootAsSummary && rawSummaryData,
+            'Store is configured to treat the root node as summary data but rawSummaryData was provided to loadData!');
+
+        if (this.haveRootSummary(rawData)) {
+            rawSummaryData = rawData[0];
+            rawData = rawSummaryData.children;
+            delete rawSummaryData.children;
```

Comment:
hmm -- we were trying not to mutate the raw data in this object.  Shouldn't we clone the rawData[0] here?